src/hostapi/jack/pa_jack.c

```diff
@@ -451,6 +452,50 @@ BlockingWaitEmpty( PaStream *s )
 
 /* ---- jack driver ---- */
 
+void replace_string(char *string, size_t buffer_length, char *to_find, char *replacement)
```

Comment:
buffer_length is ambiguous here, "buffer" is unrelated to \0 while the therm length usually excludes it.  Without const, we cannot use string literals. 
```suggestion
void replace_string(char *string, size_t string_size, const char *to_find, const char *replacement)
```

Reply:
What we are concerned with here is not the chars + `\0`, it is the size we have malloced. So I think `string_buffer_size` is better than just `string_size`.